doc/reference/efi_variables.md

```diff
@@ -0,0 +1,43 @@
+---
+discourse: ubuntu:42313
+---
+
+# EFI variables for VMs
+
+{abbr}`EFI (Extensible Firmware Interface)` variables can be used by the boot loader and the operating system to configure early system startup. <!-- Do we have some link that explains what EFI variables are? -->
+Usually, you don't need to touch them, but in specific cases they can be useful to debug EFI, SHIM, or boot loader issues in virtual machines.
+
+To configure EFI variables for a VM, use the [`lxc config uefi`](lxc_config_uefi.md) command and subcommands or the `/1.0/instances/<instance_name>/efi_vars` endpoint.
+<!-- Is there some difference between EFI and UEFI (since the command is uefi, but we're talking about EFI)? -->
```

Comment:
Basically, UEFI is just a successor of EFI. But people still saying EFI or UEFI assuming the same thing.

Reply:
Should we just use UEFI then to be consistent? We can add a "aka EFI" somewhere so that people can find the information if they are looking for EFI.